app/Jobs/DiscordAuditJob.php

```diff
@@ -164,4 +160,88 @@ public function handle(
             }
         }
     }
+
+    /**
+     * Performs audit by interating through HMS users, adding discord roles if they are missing.
+     *
+     * @param Discord $discord The discord helper object
+     * @param ProfileRepository $profileRepository Profile repository object
+     * @param RoleRepository $roleRepository Role repository object
+     *
+     * @return void
+     */
+    private function reverse(Discord $discord, ProfileRepository $profileRepository, RoleRepository $roleRepository)
+    {
+        $currentMembers = $roleRepository->findOneByName(Role::MEMBER_CURRENT)->getUsers();
+        $discordMembers = $discord->listGuildMembers();
+
+        // Build an array of roles in Discord which match roles in HMS.
+        $teams = $roleRepository->findAllTeams();
+        $auditableRoles = [
+            $discord->findRoleByName('Current Member'),
+        ];
+
+        foreach ($teams as $team) {
+            $discordRole = $discord->findRoleByName($team->getDisplayName());
+            if ($discordRole) {
+                array_push($auditableRoles, $discordRole);
+            }
+        }
+
+        Log::info('DiscordAuditJob@reverse: Identified ' . sizeof($auditableRoles) . ' roles for audit.');
```

Comment:
minor, but count is more php (size is an alias)
```suggestion
        Log::info('DiscordAuditJob@reverse: Identified ' . count($auditableRoles) . ' roles for audit.');
```